src/main/java/org/gridsuite/voltageinit/server/service/VoltageInitService.java

```diff
@@ -67,10 +75,21 @@ private static VoltageInitResult fromEntity(VoltageInitResultEntity resultEntity
     }
 
     public void deleteResult(UUID resultUuid) {
+        Optional<VoltageInitResultEntity> result = resultRepository.find(resultUuid);
+        result.ifPresent(r -> {
+            if (r.getModificationsGroupUuid() != null) {
+                networkModificationService.deleteModificationsGroup(r.getModificationsGroupUuid());
```

Comment:
Deo we have to wait this deletion before delete the result ?

Reply:
No. I replaced it with an asynchronous call